tests/phpunit/CRM/Event/BAO/CRM19273Test.php

```diff
@@ -139,19 +137,11 @@ private function contributionInvoice($contributionId) {
    * @return mixed
    */
   private function totalIncome($participantId) {
-
-    // @todo use INNER JOINS, this is not our style.
     $query = "
-      SELECT SUM(et.amount) total
-      FROM civicrm_entity_financial_trxn et
-      ,      civicrm_financial_item fi
-      ,      civicrm_line_item      li
-      WHERE  et.entity_table='civicrm_financial_item'
-      AND    fi.id = et.entity_id
-      AND    fi.entity_table='civicrm_line_item'
-      AND    fi.entity_id = li.id
-      AND    li.entity_table = 'civicrm_participant'
-      AND    li.entity_id = ${participantId}
+      SELECT SUM(fi.amount) total
+      FROM civicrm_financial_item fi
```

Comment:
@eileenmcnaughton we don't need the `civicrm_entity_financial_trxn`?

Reply:
I think that joins the financial_trxn tables but not the financial_items (which join directly).